src/frontend/apps/desk/src/api/fetchWrapper.ts

```diff
@@ -0,0 +1,42 @@
+import useAuthStore from '@/auth/useAuthStore';
+
+interface FetchWrapperOptions {
+  withoutProxy?: boolean;
+}
+
+export const fetchWrapper = async (
+  input: RequestInfo,
+  init?: RequestInit,
+  options?: FetchWrapperOptions,
+): Promise<Response> => {
+  /**
+   * API Helper
+   */
+  if (
+    !options?.withoutProxy &&
+    typeof input === 'string' &&
+    input.startsWith('/api')
+  ) {
+    const url = `${process.env.NEXT_PUBLIC_API_URL}${input.replace(
+      '/api/',
```

Comment:
Not a huge fan of manipulating `/api` and `/api/` literals, this might be error-prone. Let's keep that way, but I think we should improve this mechanism

Reply:
It is not top yes, the api url are like that `http://localhost:8071/api/v1.0/` (http://localhost:8071/v1.0/swagger/), if we can have this base by example `http://localhost:8071/v1.0/api/`, we will not need to rewrite the url but just have add a api baseUrl.

Reply:
I see, might be great actually, let's discuss it on Monday stand-up, could you open an issue?

Reply:
To have the versioning I understand, but I don't think there is anything wrong to swap from `http://localhost:8071/api/v1.0/` to `http://localhost:8071/v1.0/api/`, except all the backend tests would have to be updated apparently 😃 

Don't need 2 wrappers, if you don't need the wrapper, nothing block you to just use `fetch`.
I improve readability: https://github.com/numerique-gouv/people/pull/13/commits/4ed61c68502549132492abbe2ca20a506533c182

Reply:
From our conversation: [4c59ed8](https://github.com/numerique-gouv/people/pull/13/commits/4c59ed863e8a62514fc9987021a6ac3327cbf2c8)